src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Athletes/AthleteForm.razor

```diff
@@ -0,0 +1,51 @@
+﻿@using NTS.Domain.Objects
+@using NTS.Domain.Setup.Entities;
+@using NTS.Judge.Blazor.Contexts
+@using Not.Blazor.Forms;
+@using Not.Blazor.TM.Forms;
+@using Not.Blazor.TM.Forms.Components;
+
+@inherits NotForm<Athlete>;
+
+<EditForm Model="_model">
+    <MudTextField @ref="_nameField" @bind-Value="_model.Name" Label="@Localizer.Get("Name")" />
+    <MudTextField @bind-Value="_model.FeiId" Label="Fei Id" HelperText="@Localizer.Get("Leave empty if horse doesn't compete by FEI rules")" Variant="Variant.Text" Margin="Margin.Dense" />
```

Comment:
_Leave empty if horse doesn't compete by FEI rules_ isn't exactly correct as it has nothing to do with the actual rules of the event. I think _Required for FEI export_ would be better.

Furthermore what happens if we decide to change that text later on. Isn't there a better approach if it's going to be repeated in multiple places?

Reply:
What about my question?